docs/generated/packages/vite.json

```diff
@@ -20,26 +20,65 @@
       "factory": "./src/generators/init/init",
       "schema": {
         "cli": "nx",
-        "title": "Add Vite Configuration to the workspace",
-        "description": "Add Vite Configuration to the workspace.",
+        "title": "Initialize Vite in the workspace.",
+        "description": "Initialize Vite in the workspace.",
         "$id": "init-vite-plugin",
         "type": "object",
         "properties": {
           "uiFramework": {
             "type": "string",
             "description": "UI Framework to use for Vite.",
-            "enum": ["react", "vue3", "vue3-jsx"],
+            "enum": ["react", "vue3", "vue3-jsx", "vanilla"],
```

Comment:
If you are adding vue here. Have you considered adding solidjs vite-plugin-solid? It would mean we can remove solid implementation in nxext